modules/update-validating-webhook-configuration.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_content-type: PROCEDURE
+[id="update-validating-webhook-configuration_{context}"]
+= Update ValidatingWebhookConfiguration
+
+[role="_abstract"]
+Previous {product-title-short} versions included a wrong entry in the ValidatingWebhookConfiguration. To fix it, you must update the ValidatingWebhookConfiguration.
+
+.Procedure
+
+* If you have enabled `listenOnEvents` in your admission controller, you must run the following command:
```

Comment:
ptal >> I see instances  of admission controller vs Admission Controller e.g.

1.https://docs.openshift.com/acs/3.72/installing/installing_helm/install-helm-quick.html
2. https://docs.openshift.com/acs/3.72/operating/use-admission-controller-enforcement.html

Reply:
Thank you @Amrita42 I've fixed all instances and changed them to "Admission controller". Capital A because we use it as a component of RHACS different from admission controller in general. There are a few instances of "Admission Controller" where it appears in the UI text.